_articles/before-you-open-a-support-ticket.md

```diff
@@ -0,0 +1,189 @@
+
+### 2. Do you have a Live USB drive with a copy of either Ubuntu or PopOS available in case we need to use it as a rescue disk? 
+
+This will allow us an outside OS environment to access your installed system, potentially rescue files that have not been backed up, or quickly reinstall the OS if necessary. 
+
+We have a help article on creating Live USBs which can be found here: https://support.system76.com/articles/live-disk/ 
+
+## Software
+
+### 3. Have you tested in either the Recovery Partition or a Live USB to see if the issue is present there? 
+
+We have help articles on the Recovery Partition and how to access the Boot Menu listed below.
+
+Recovery Partition: https://support.system76.com/articles/pop-recovery/ 
+
+Boot Menu: https://support.system76.com/articles/boot-menu/ 
+
+
+### 4. Is your system fully up to date?
+
+NOTE: The following instructions can be used in either Ubuntu or PopOS.
+
+You can open the terminal by clicking Activities in the upper-left corner and searching for "Terminal", or by pressing <kbd> SUPER </kbd>
+When you run the first command with "sudo" it will ask you to enter your password. 
+Typing your password will not show anything in the Terminal window. That is normal and for security. 
+Just enter your password as you normally would and then hit enter. You'll also need to hit enter after each command.
+
+```bash
+sudo apt clean
+sudo apt update -m
+sudo dpkg --configure -a
+sudo apt install -f
+sudo apt dist-upgrade
+sudo apt autoremove --purge
+reboot
+```
+
+Then let's run:
+
+```bash
+sudo apt update
+sudo apt full-upgrade
+reboot
+```
+
+Source: https://support.system76.com/articles/package-manager-pop/ 
+
+### 5. Have you tried creating an administrative Test User account to see if the issue is present in a different user?
+
+That can be done by:
+
+- Navigate to Settings>Users
+- Click the Unlock button
+- Click the  "add user" button that appears.
+- Create a user with whatever name and credentials you want.
+- Make sure click the "Administrator" option for the user.
+- Reboot and log into the new user.
+
+Source: https://support.system76.com/articles/other-accounts/ 
+
+### 6. Is there a new OS version available? 
+Upgrading to the latest OS version usually includes newer kernel modules and software packages as well as security updates and bug-fixes. 
+These updates may directly or indirectly resolve your issue.
+
+Upgrade Pop: https://support.system76.com/articles/upgrade-pop/ 
+
+Upgrade Ubuntu: https://support.system76.com/articles/upgrade-ubuntu/ 
+
+### 7. Have you tried reinstalling the packages or programs that are giving you trouble? 
+
+We are happy to provide instructions on how to do that if need be. Just let us know which program(s) are misbehaving and we can explain how to uninstall it/them.
+
+Typically the Terminal commands to reinstall a program are:
+
+```bash
+sudo apt remove <packagename>
+```
+OR:
+
+```bash
+flatpak uninstall <packagename>
```

Comment:
I think it is 'flatpak remove` ?

Reply:
It's flatpak uninstall according to this: https://www.systutorials.com/docs/linux/man/1-flatpak-uninstall/